### DISCUSSION TITLE
建议添加从语音文件生成字幕的功能

### DISCUSSION THREAD

**Discussion by yu-xiaowen** [immersive-translate/immersive-translate#981] [Ideas]
### 新功能 | New Feature

用语音文件生成字幕文件

### 为什么您想要添加这些功能 | Why you want to add these features

youtube自带英文字幕断句有时很糟糕，一句话拆成好几段，从英文字幕文件翻译成中文字幕文件效果很差。谷歌自带翻译也不够好。

### 提议 | Proposal

从本地选取语音文件，生成英文和中文翻译的字幕文件。如果直接输入网址，自动提取语音更好。

**Comment by theowenyoung** [comment]
这是一个大活，未来可能支持。
